src/Impostor.Server/Net/Client.cs

```diff
@@ -260,8 +259,8 @@ public override async ValueTask HandleMessageAsync(IMessageReader reader, Messag
 
                 case MessageFlags.GetGameListV2:
                 {
-                    Message16GetGameListC2S.Deserialize(reader, out var options, out _, out _, out var filterOptions);
-                    await OnRequestGameListAsync(options, filterOptions);
+                    Message16GetGameListC2S.Deserialize(reader, out _, out _, out _, out _);
```

Comment:
why even deserialize if you're going to throw the message in the trash and disconnect them anyway?